src/openforms/forms/templates/admin/forms/formsexport/email_content.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+
+{% blocktrans %}
+Hi,<br>
+<br>
+Your zip file containing the exported forms is ready and can be downloaded at the following URL:<br>
+{{ download_url }}<br>
+<br>
+
+Note: the URL is only valid once.
+
+<br>
+Best wishes,<br>
+<br>
+Open Forms<br>
+{% endblocktrans %}
```

Comment:
See earlier remark about the `blocktrans` contents. I think in the next review/iteration we'll have to check the copy of the email a bit better too.